common/smoke-tests/pom.xml

```diff
@@ -126,13 +126,13 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-identity</artifactId>
-      <version>1.0.3</version>
+      <version>1.1.0</version>
```

Comment:
@danieljurek Should we add Jim's version tags to these so when we release, it doesn't break?